### PR TITLE
Desktop portfolio 

### PR DESCRIPTION
**Pull request by yahyaelganyni1**
transforming the mobile virgin into desktop

### CODE REVIEW

**Review by vichuge — CHANGES_REQUESTED** [review]
# Project invalidated 🚫 

Hi @yahyaelganyni1, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

cat_massage

## Why does this happen ❓ ❓ ❓ 

You have submitted a project, that has problems with Linters errors
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -264,30 +274,29 @@ <h3>
         ></textarea>
         <button id="btn" type="submit">Get in touch</button>
       </form>
```
Comment (Shaher-11) [inline index.html:285]: Kindly stick to the given design in this section as shown below
### Original design
right
### Your project 
wrong

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @yahyaelganyni1,

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comment below.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html:285]: Well, addressed the issue!

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved✔️🎉 🎉 
Hi @yahyaelganyni1, 

Your project is complete! I am just like wow!! you did an amazing job anyone can see a lot of hard work in your project!!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional
- [ ] There is a lot of empty space between the headline container section and a main section on the desktop you can reduce it
- [ ] Check this mobile version
hy
 you can do more here to make it looks hotter😊😊 even on tablet view.. but besides that well done!

### Cheers and Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -15,3 +15,7 @@
 ## project sample
 
 ![screenshot](./imges/Screenshot.png)
+
+## online version 
+
+- [portflio](https://yahyaelganyni1.github.io/mobile-version-skeleton/)
```
Comment (teekaytech) [inline README.md]: I think it's high time you update your Readme file: 
- Your portfolio is no more a mobile version, but a full version. Therefore, I suggest you update the project title and description on the readme :blush:  
- Also, it is a good practice to always include the `Getting Started` section on the Readme. This section highlights the steps involved in setting up your project for any visitor. [This template](https://github.com/microverseinc/readme-template) might help.
- Another important part of the Readme is the `Built With` section. This is where you highlight tools, languages, and technologies used to build the project. Kindly add it :bowing_man:

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @yahyaelganyni1 :wave:

**Great work on deploying your project 👏** 

__However, you need to make some required changes before you can merge.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes :recycle:**

- [ ] You've used the branch & PR raised for the `desktop version` milestone. Since this is a different feature, you're expected to create another branch for this milestone, implement the necessary features, and create a new PR using the `main` branch as the base branch. Kindly fix this.
- [ ] Check for the inline comments below. 

### **Optional Changes** :zap: 
N/A

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

*Feel free to leave any questions or comments in the PR thread if something is not 100% clear.*

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

Re-reviewed by: [**Taofeek Olalere**](https://github.com/teekaytech)
*Cheers and Happy Coding* :smiley:

**Pull request merged**